packages/core/components/VisxShape.tsx

```diff
@@ -21,23 +17,41 @@ interface VisxShapeProps {
   left: number
   top: number
   display: boolean
-  index: GlyphKeys
+  index: number
   stroke: string
   key: string | number
+  strokeWidth?: number
 }
 
-const VisxShape: React.FC<VisxShapeProps> = ({ fill, size, left, top, display, index, stroke, key }) => {
+const VisxShape: React.FC<VisxShapeProps> = ({
+  fill,
+  size,
+  left,
+  top,
+  display,
+  index,
+  stroke,
+  key,
+  strokeWidth = 0
+}) => {
   if (!display) {
-    return null
+    return <g></g>
   }
 
   const Shape = glyphs[index]
   const isFlipped = index === 4
+
   const transform = `translate(${left}, ${top}) ${isFlipped ? 'rotate(180)' : ''}`
 
   return (
     <g key={key} transform={transform}>
-      <Shape strokeWidth={0} fill={fill} stroke={stroke} size={size} />
+      {index === 6 ? (
+        <Text textAnchor='start' fill={fill} dy={3.5} dx={-5} fontSize={size / 4}>
```

Comment:
This shape should be added as CustomGlyph from Visx. Then you can include it in the Glyphs array and remove the logic in this section:

'pentagon': CustomGlyph

Reply:
I am not sure if Visx supports custom glypth. how ?

Reply:
I think you can follow the example in this file:
https://airbnb.io/visx/glyphs

The import is Glyph as CustomGlyph